models/grafana_helper.go

```diff
@@ -67,6 +67,9 @@ func (g *GrafanaClient) Validate(ctx context.Context, BaseURL, APIKey string) er
 }
 func (g *GrafanaClient) makeRequest(ctx context.Context, queryURL, APIKey string) ([]byte, error) {
 	req, err := http.NewRequest(http.MethodGet, queryURL, nil)
+	if err != nil {
```

Comment:
The grafana sdk isn’t great. We’ll want to test to confirm behavior remains the same here.

Reply:
I have just captured the error which was neither captured nor reported. This wont have any effect on the sdk.

Reply:
Oops. I was on my mobile phone reviewing this PR and thought that you had removed, not added this catch. 

Ok, 👍